templates/rhpam73-authoring-ha.yaml

```diff
@@ -295,6 +295,23 @@ parameters:
   name: ES_VOLUME_CAPACITY
   value: 1Gi
   required: true
+## OpenShift Enhancement BEGIN
+- displayName: Enable OpenShift Integration
+  description: If set to true, turns on OpenShift integration feature (Sets the org.kie.workbench.controller.openshift.enabled system property)
+  name: KIE_CONTROLLER_OCP_ENABLED
+  value: "true"
```

Comment:
Does this mean that authoring templates will now have OpenShift integration enabled by default?

Reply:
It wasn't intended, is changed to 'false'.